Mayhemfile

```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/rishabh1024/ffproxy-fuzz:latest
```

Comment:
Please omit the image name here, as the image name gets populated during the workflow run.

Reply:
Completed.